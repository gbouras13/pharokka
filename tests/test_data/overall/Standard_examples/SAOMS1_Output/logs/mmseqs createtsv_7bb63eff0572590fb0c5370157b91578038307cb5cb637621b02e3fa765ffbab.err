Command line: mmseqs createtsv ../../databases/phrogs_profile_db SAOMS1_Output/target_dir/target_seqs SAOMS1_Output/mmseqs/results_mmseqs SAOMS1_Output/mmseqs_results.tsv --full-header --threads 8
