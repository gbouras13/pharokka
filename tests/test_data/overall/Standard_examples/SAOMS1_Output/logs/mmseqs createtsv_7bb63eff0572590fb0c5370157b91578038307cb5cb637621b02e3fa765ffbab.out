createtsv ../../databases/phrogs_profile_db SAOMS1_Output/target_dir/target_seqs SAOMS1_Output/mmseqs/results_mmseqs SAOMS1_Output/mmseqs_results.tsv --full-header --threads 8 

MMseqs Version:                 	13.45111
First sequence as representative	false
Target column                   	1
Add full header                 	true
Sequence source                 	0
Database output                 	false
Threads                         	8
Compressed                      	0
Verbosity                       	3

Time for merging to mmseqs_results.tsv: 0h 0m 0s 23ms
Time for processing: 0h 0m 0s 44ms
